src/gui/widgets/listbox.cpp

```diff
@@ -1095,16 +1094,14 @@ widget* builder_horizontal_listbox::build() const
 	list_view* widget = new list_view(
 			true, true, generator_base::horizontal_list, true, list_builder);
 
-	init_control(widget);
+	//init_control(widget);
```

Comment:
The work of init_control is now spread between the widget and styled_widget ctors.